lib/utils.js

```diff
@@ -56,9 +43,9 @@ utils.isPath = function isPath(pattern, opts) {
  * @return {Function}
  */
 
-utils.hasPath = function hasPath(pattern, opts) {
+utils.pathContains = function(pattern) {
   return function(fp) {
-    return utils.unixify(pattern, opts).indexOf(fp) !== -1;
```

Comment:
Is `unixify` being dropped in these two utils because it's happening in `matcher()` now instead? 

I have to admit that messing with the `unixify` treatment makes me a little nervous considering issues we've wrestled with in the past.